inst/input/hector_ssp119.ini

```diff
@@ -130,22 +130,25 @@ CH4_emissions=csv:tables/ssp119_emiss-constraints_rf.csv     ; emissions time se
 NOX_emissions=csv:tables/ssp119_emiss-constraints_rf.csv     ; emissions time series
 CO_emissions=csv:tables/ssp119_emiss-constraints_rf.csv      ; emissions time series
 NMVOC_emissions=csv:tables/ssp119_emiss-constraints_rf.csv   ; emissions time series
-H2_emissions=csv:tables/ssp119_emiss-constraints_rf.csv      ; emissions time series
+H2_emissions[1745]=0                                         ; assumed 0 emissions for the RCMIP era SSPs
 
 
 TOH0=6.6			; inital OH lifetime (years)
 CNOX=0.0042			; coefficient for NOX
 CCO=-0.000105		; coefficient for CO
 CNMVOC=-0.000315	; coefficient for NMVOC (non methane VOC)
 CCH4=-0.32			; coefficient for CH4
-CH2=-0.00044625 ; coefficient for H2 TODO-H2
+H2_CCO=4.25         ; adjust the CO coefficient for H2 emissions effect on OH lifetime H2 see PR 787
```

Comment:
Why does this say "adjust the"? Is this a record of the change, or a suggestion for users? A bit unclear

Reply:
<img width="519" height="44" alt="472681803-33326e7a-33ff-4c4f-98cf-800f07280cdd" src="https://github.com/user-attachments/assets/71560c84-b5d1-43c8-b2d1-440ebf840b26" />

Maybe the adjust is not the correct term, see equation here, we don't have a direct H2 emissions effect coefficient, instead we scale or adjust the CO coefficient

Reply:
Oh, thank you. Maybe `; scaling factor for the CO coefficient that affects OH lifetime; see PR787`